clients/src/main/java/org/apache/kafka/common/record/ControlRecordUtils.java

```diff
@@ -27,6 +29,8 @@
 public class ControlRecordUtils {
 
     public static final short LEADER_CHANGE_SCHEMA_VERSION = new LeaderChangeMessage().highestSupportedVersion();
+    public static final short METADATA_SNAPSHOT_HEADER_VERSION = new MetadataSnapshotHeaderRecord().highestSupportedVersion();
```

Comment:
Do we really need this constant? It seems confusing since it implies there is only one version ever, but there may well be more versions later on. If you mean highest supported, why not just say highest supported rather than referring to "the version"?

Reply:
I honestly do not have an opinion here. Was following what was done for the LEADER_CHANGE message (right above it). I think saying highest supported version (or maybe even current version) would make sense. I would then change the leader change message related constants as well to be consistent. Let me do that in the next version.